src/realm/object-store/c_api/types.hpp

```diff
@@ -51,54 +52,21 @@ struct DuplicatePrimaryKeyException : std::logic_error {
 struct InvalidPropertyKeyException : std::logic_error {
     using std::logic_error::logic_error;
 };
-struct CallbackFailed : std::runtime_error {
+*/
```

Comment:
can we remove the commented out section above?

Reply:
Yes